cmd/agent/app/jmx.go

```diff
@@ -191,6 +191,7 @@ func loadConfigs() {
 		if check.IsJMXConfig(c.Name, c.InitConfig) && (includeEverything || configIncluded(c)) {
 			fmt.Println("Config ", c.Name, " was loaded.")
 			jmx.AddScheduledConfig(c)
+			runner.ConfigureCheck(c.InitConfig)
```

Comment:
any reason you only call `ConfigureCheck` here instead of both `ConfigureCheck` and `ConfigureInstance`?

Reply:
Wasn't sure it was worth iterating on instances, done now.